DeploymentAgentReconfigurationScripts/DetachAttachCollectionScenario/AzureDevOpsServer2019/ConfigureAgentToTargetServer.ps1

```diff
@@ -0,0 +1,140 @@
+<# 
+This script is applicable for AzureDevOpsServer 2019 or later.
+This is a guidance script to re-configure deployment group agents to the target TFS after detach/attach activity. This script is applicable if the source collection is not online. If source TFS collection is online (Side by Side scenario), use ConfigureAgentToTargetServerUsingSourceServer,ps1 script instead.
+
+Inputs:
+1. targetTFSUrl : New TFS instance url.
+2. targeTFSPatToken : PAT token from target TFS with Deployment group manage scope (at least)
+3. existingAgentFolder: The script will Auto-detect the agent folder if it was running as windows service. User need to pass the folder path otherwise.
+4. agentDownloadUrl: this is optional, user can specify if she wants any specific agent version to be installed.
+5. tags : Tags of the target. Since the source TFS is not available, user need to pass the tags explicitly.
+5. action: By default, the script will not do any update operation, it will just print what changes will happen after the script is applied. User need to set action parameter to 'apply' to update execute actual steps.
+
+Output:
+If action is set to apply, this script this script will
+1. Configure a new agent to the deployment group in the target TFS with same properties as the existing agent in the source TFS including tags.
+Example usage:
+.\ConfigureAgentToTargetServer.ps1 -targetTFSUrl <newTfsUrl> -sourceTFSPatToken <PAT-for-old-TFS>
+.\ConfigureAgentToTargetServer.ps1 -targetTFSUrl <newTfsUrl> -sourceTFSPatToken <PAT-for-old-TFS> -existingAgentFolder <AgentFoilder (C:\vstsagents\A1)> -tags 'Web,DB'
+.\ConfigureAgentToTargetServer.ps1 -targetTFSUrl <newTfsUrl> -sourceTFSPatToken <PAT-for-old-TFS> -existingAgentFolder <AgentFoilder (C:\vstsagents\A1)> -action 'apply'
+#>
+
+param([string]$targetTFSUrl,
+      [string]$targetTFSPatToken,
+      [string]$existingAgentFolder = "",
+      [string]$agentDownloadUrl = 'https://vstsagentpackage.azureedge.net/agent/2.141.1/vsts-agent-win-x64-2.141.1.zip',

```

Comment:
Is this the agent version we are shipping for 2019? I presume this version will change as we get closer to RTM.